route.go

```diff
@@ -25,15 +40,36 @@ type Route struct {
 	Type       int
 	Tos        int
 	Flags      int
+	MPLSDst    int
```

Comment:
I believe 0 is a valid mpls label id, so it might be necessary to use an *int here?